integration_tests/e2e/induction/createInduction.cy.ts

```diff
@@ -474,7 +474,7 @@ context('Create an Induction', () => {
     //  Highest Level of Education: SECONDARY_SCHOOL_TOOK_EXAMS
     //  Qualifications:
     //    Pottery, C, Level 4
-    cy.task('stubGetEducation', prisonNumberForPrisonerWithNoInduction)
+    cy.task('stubGetEducation', { prisonNumber: prisonNumberForPrisonerWithNoInduction })
```

Comment:
The `stubGetEducation` stub has changed very slightly , as it now takes an options object as it's arg, rather than simply the prison number. The options object is used to control whether there are any qualifications in the education record or not.